apps/spoolman/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.8'
+services:
+  spoolman:
+    image: ghcr.io/donkie/spoolman:0.16.1
+    restart: unless-stopped
+    volumes:
+      - ${APP_DATA_DIR}/data:/home/app/.local/share/spoolman 
+    ports:
+      - ${APP_PORT}:8000
+    environment:
+      - TZ=${TZ}
+      - PUID=1000
+      - PGID=1000
```

Comment:
The app has issues running with 1000, 1000 

```usermod: no changes
User UID: 1000
User GID: 1000
Starting uvicorn...
INFO:     Started server process [1]
INFO:     Waiting for application startup.
spoolman.env               WARNING  Data directory is not writable, trying to fix it...
chown: /home/app/.local/share/spoolman: Operation not permitted
chown: /home/app/.local/share/spoolman: Operation not permitted
spoolman.env               ERROR    Data directory is not writable. Please run "sudo chown -R 1000:1000 /path/to/spoolman/datadir" on the host OS.
ERROR:    Traceback (most recent call last):
  File "/home/app/spoolman/.venv/lib/python3.11/site-packages/starlette/routing.py", line 677, in lifespan
    async with self.lifespan_context(app) as maybe_state:
  File "/home/app/spoolman/.venv/lib/python3.11/site-packages/starlette/routing.py", line 566, in __aenter__
    await self._router.startup()
  File "/home/app/spoolman/.venv/lib/python3.11/site-packages/starlette/routing.py", line 654, in startup
    await handler()
  File "/home/app/spoolman/spoolman/main.py", line 70, in startup
    env.check_write_permissions()
  File "/home/app/spoolman/spoolman/env.py", line 369, in check_write_permissions
    sys.exit(1)
SystemExit: 1

ERROR:    Application startup failed. Exiting.
```

so please remove the completely so the app runs as root.